includes/class-wp-smime.php

```diff
@@ -141,6 +141,12 @@ public static function encrypt ( $message, $headers, $certificates ) {
 
         if ( $smime ) {
             $parts = explode( "\n\n", $smime, 2 );
+
+            // add new mime type for improved compatibility (e.g. Roundcube)
+            // https://tools.ietf.org/html/rfc3851
+            if ( false === stripos( $parts[0], "application/pkcs7-mime" ) ) {
+                $parts[0] .= "\nContent-Type: application/pkcs7-mime;\n";
```

Comment:
The trailing newline at the end of the header string may be harmless but will cause future operations that `join( "\n\n", … )` the email headers with the email body to end up with an email body whose first character is always `\n`. This technically changes the content of the message as is passed to our routine, which is something I'd like to avoid.

Please ammend this commit to omit the trailing newline in `$parts[0]`. Thanks!